content/docs/guides/continuous-delivery/azure-devops.md

```diff
@@ -99,7 +99,10 @@ The following are samples only. You may choose to structure your configuration a
 
 ### Pulumi Task Extension for Azure Pipelines
 
-We built a task extension that will let you easily use Pulumi in your CI/CD pipelines. It can be used with the Azure Pipelines wizard UI or the YAML config. The task handles installing the Pulumi CLI, and running any commands without the need for any scripts.
+We built a task extension that will let you easily use Pulumi in your CI/CD pipelines. It can be used with the Azure Pipelines wizard UI or the YAML config.
```

Comment:
```suggestion
We built a task extension that lets you easily use Pulumi in your CI/CD pipelines. It can be used with the Azure Pipelines wizard UI or the YAML config.
```